### PR TITLE
Update iotagent.apib

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: doc/apiary/iotagent.apib [hunk 1]
```diff
@@ -93,9 +93,9 @@ Mandatory fields are identified in every operation.
 
 
 
-### Retrieve a service [GET]
+### Retrieve a service group [GET]
```
Comment (fgalan) [inline doc/apiary/iotagent.apib]: The "cannonical name" is "configuration group", isn't it? At least that is the one used along https://thinking-cities.readthedocs.io/en/master/device_api/index.html, as far as I understand.

Comment (AlvaroVega) [reply doc/apiary/iotagent.apib]: fixed in ad54e4edc563568e1539f201ad40bb3a3154f100

File: doc/apiary/iotagent.apib [hunk 2]
```diff
@@ -93,9 +93,9 @@ Mandatory fields are identified in every operation.
 
 
 
-### Retrieve a service [GET]
+### Retrieve a configuration group [GET]
 
-With Fiware-ServicePath you can retrieve a subservice or all subservices.
+Retrieve a group configuration.
```
Comment (fgalan) [inline doc/apiary/iotagent.apib]: group configuration -> configuration group

File: doc/apiary/iotagent.apib [hunk 3]
```diff
@@ -71,7 +71,7 @@ For example, `http://127.0.0.1:8080/iot/`.
 # Group Configuration API
 
 ## Services [/services{?limit,offset,resource,apikey,device}]
-Services are the higher level in IoT Platform. When you manipulate a service, you use a Fiware-Service header with its name. Parameters apply to different operations.
+Group configuration for iotagents.
```
Comment (fgalan) [inline doc/apiary/iotagent.apib]: group configuration -> configuration group

File: doc/apiary/iotagent.apib (hunk 2 above)
Comment (AlvaroVega) [reply doc/apiary/iotagent.apib]: fixed in 8ba00f67db1897dddfea5c1bc64bdd0d9ac18aeb

File: doc/apiary/iotagent.apib (hunk 3 above)
Comment (AlvaroVega) [reply doc/apiary/iotagent.apib]: fixed in 8ba00f67db1897dddfea5c1bc64bdd0d9ac18aeb

**Pull request merged**